xml/System.Security.Cryptography/SHA256.xml

```diff
@@ -112,7 +110,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>Creates an instance of the default implementation of <see cref="T:System.Security.Cryptography.SHA256" />.</summary>
+        <summary>Creates an instance of the default implementation of <see cref="T:System.Security.Cryptography.SHA256" />. When this behavior is not overriden, the default implementation is <see cref="T:System.Security.Cryptography.SHA256Managed" />.</summary>
         <returns>A new instance of <see cref="T:System.Security.Cryptography.SHA256" />.</returns>
         <remarks>To be added.</remarks>
         <exception cref="T:System.Reflection.TargetInvocationException">The algorithm was used with Federal Information Processing Standards (FIPS) mode enabled, but is not FIPS compatible.</exception>
```

Comment:
The exception under FIPS mode was removed in .NET Framework 4.6.2 (and doesn't apply at all to .NET Core).  See previous comment about the default for the Create() method.

Reply:
Thanks @bartonjs !

@rpetrusha Same question as above : How should I distinguish .NET Core, .NET Framework < 4.6.2 and .NET Core > 4.6.2 ?

Reply:
Change "The algorithm..." to "On the .NET Framework 4.6.1 and earlier versions only: The algorithm..."

Reply:
@rpetrusha Thanks !